### ISSUE TITLE
Windows install fails (hangs) if Git is not installed

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#2961]
rc2 - This test was run on a 32 bit machine, but the same thing happens on 64 bit Windows.
Installer warns that Git is required and prompts user to click OK to install it, but then this message is displayed:
![image](https://user-images.githubusercontent.com/8171759/31665533-2130bf40-b317-11e7-9267-788ee5988dde.png)
After the message is dismissed (actually it is displayed twice), the tC Loading panel is displayed as is normal in the installation, but it never finishes loading.

**Comment by cckozie** [comment]
alpha.11 on Win10 64 bit
Uninstalled previous installation of tC and deleted related folders. Also uninstalled Git. 
Ran the 64 bit install which installed both tC and Git, but at the end was presented with this dialog:
![git1](https://user-images.githubusercontent.com/8171759/32985392-5e5bf606-cc7f-11e7-9237-81f320f6b7d7.PNG)

After dismissing that dialog I was presented with this one:
![git2](https://user-images.githubusercontent.com/8171759/32985402-749d45d2-cc7f-11e7-8755-5418103638f2.PNG)

After dismissing that dialog, the Welcome to tC...Loading dialog came up. I let it run for more than 30 minutes before killing it. At that point the setup icon was still showing in the task bar. I had to open task manager to kill it.
After that I double clicked the tC desktop icon and tC did start and run properly.
